rpc/clients/typescript/src/ws_client.ts

```diff
@@ -278,17 +279,19 @@ export class WSClient {
      * @param perPage number of signedOrders to fetch per paginated request
      * @returns all orders, their hash and their fillableTakerAssetAmount
      */
-    public async getOrdersAsync(perPage: number = 200): Promise<OrderInfo[]> {
+    public async getOrdersAsync(perPage: number = 200): Promise<GetOrdersResponse> {
         let snapshotID = ''; // New snapshot
 
         let page = 0;
-        const getOrdersResponse: GetOrdersResponse = await this._wsProvider.send('mesh_getOrders', [
+        const rawGetOrdersResponse: RawGetOrdersResponse = await this._wsProvider.send('mesh_getOrders', [
             page,
             perPage,
             snapshotID,
         ]);
-        snapshotID = getOrdersResponse.snapshotID;
-        let ordersInfos = getOrdersResponse.ordersInfos;
+        snapshotID = rawGetOrdersResponse.snapshotID;
```

Comment:
Can we move this conversion to a helper function/method? The type conversions in this file (and in the __browser/__ code) are one of the things we need to improve test coverage for and this would make it easier to test. The previous code didn't really do any conversion.